web/src/redux/apis/endpoints/Setting.endpoint.ts

```diff
@@ -63,6 +64,15 @@ const ConfigEndpoint = (builder: TTestApiEndpointBuilder) => ({
     providesTags: () => [{type: TracetestApiTags.SETTING, id: ResourceType.AnalyzerType}],
     transformResponse: (rawLinter: TRawLinter) => Linter(rawLinter),
   }),
+  getTestRunner: builder.query<TestRunner, unknown>({
```

Comment:
I was thinking we can simplify the endpoints for settings and the overall resources, we can think about a clever way to handle that in the future

Reply:
Like that!